packages/next-app/src/app/daoadmin/action/[actionId]/(components)/ActionAttributeContent.tsx

```diff
@@ -76,12 +122,18 @@ export function ActionAttributeContent() {
               <div className="flex flex-col gap-4">
                 {selectedAction?.fields.map((field) => {
                   return (
-                    <Input
-                      key={field.name}
-                      label={field.name}
-                      placeholder={field.placeholder}
-                      {...register(toSlug(`${field.name}`))}
-                    />
+                    <div key={field.name}>
+                      <Input
+                        label={field.name}
+                        placeholder={field.placeholder}
+                        {...register(toCammelCase(`${field.name}`))}
+                      />
+                      <CheckPoolID
+                        fieldName={field.name}
+                        poolName={poolName}
+                        poolError={poolError}
+                      />
```

Comment:
any reason to abstract this into a component? I don't see it's abstracting any functionality that other components could use going up and down to check conditions made the code harder to reason about imo